Assets/Scripts/Wall/WallSegment.cs

```diff
@@ -22,71 +22,74 @@ public class WallSegment : MonoBehaviour
         public bool chosenOne;
 
         private MeshFilter _meshFilter;
+
         private void Start()
         {
             _meshFilter = GetComponentInChildren<MeshFilter>();
             normalWall = _meshFilter.mesh;
             // UpdateSoldierState();
         }
 
-        public void RepairWall()
```

Comment:
RepairWall should return whether or not the repairing was successful. If the wall was already full health it should tell the player so.